python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -650,6 +650,54 @@ def moe_align_block_size_triton(
     )
 
 
+@triton.jit
+def init_sorted_ids_and_cumsum_buffer_kernel(
+    sorted_ids_ptr,
+    cumsum_buffer_ptr,
+    max_num_tokens_padded: tl.constexpr,
+    topk_ids_numel: tl.constexpr,
```

Comment:
I wonder whether using max_num_tokens_padded and topk_ids_numel will create too many kernels without yielding significant performance benefits

Reply:
Right, if we need to consider prefill, does that mean we can only resolve it with a CUDA kernel?

Reply:
Make sense, I can set `ALIGNED_NUM_EXPERTS_P1` to 512?

Reply:
> Make sense, I can set `ALIGNED_NUM_EXPERTS_P1` to 512?

triton.next_power_of_2(num_experts + 1)